openbb_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -134,6 +137,36 @@ def print_help(self):
         console.print(text=mt.menu_text, menu="Cryptocurrency - Onchain")
 
     @log_start_end(log=logger)
+    def call_dquery(self, other_args: List[str]):
```

Comment:
we don't have a public API to get the query definition yet.
this might work, but it might also not work due to bot protection.
just a heads up